compiler/expressions.go

```diff
@@ -387,7 +387,7 @@ func (fc *funcContext) translateExpr(expr ast.Expr) *expression {
 				switch basic.Kind() {
 				case types.Int32, types.Int:
 					return fc.formatParenExpr("$imul(%e, %e)", e.X, e.Y)
-				case types.Uint32, types.Uintptr:
+				case types.Uint32, types.Uint, types.Uintptr:
```

Comment:
Good idea, I have a test from the crypto debugging that I think I can whittle down to a simple test. (I have a 9 month old baby, so I totally understand 😆)

Reply:
Added a unit-test that shows how the multiplication of the 32-bit values result the same. If we don't add `types.Uint` it will result in different values than `types.Uint32`.